karate-core/src/main/java/com/intuit/karate/core/ScriptBridge.java

```diff
@@ -423,7 +423,7 @@ public Object xmlPath(Object o, String path) {
             }
         }
         ScriptValue sv = Script.evalXmlPathOnXmlNode((Node) o, path);
-        return sv.getValue();
+        return null != sv ? sv.getValue() : "";
```

Comment:
did you try returning null instead of ""

also I personally prefer `return sv == null ? "" : sv.getValue()`

Reply:
yup - we can return null too - the failing tests will show actual is null vs expected. If we leave to return empty string instead - then it will be actual "" vs expected - which I think is more inline. Let me know and will change accordingly. Thank you.

Reply:
go with null - because I think somewhere else we take it to mean "not present", anyway I think that logic is improved. can you add a test here also to see how it behaves: https://github.com/intuit/karate/blob/rewrite/karate-core2/src/test/java/com/intuit/karate/match/MatchTest.java

Reply:
made code change per above. will make change to rewrite branch and update. Thank you.